client/me/help/main.jsx

```diff
@@ -116,106 +115,109 @@ class Help extends React.PureComponent {
+				<div className="help__support-links">
+					{ this.getCoursesTeaser() }
+					<CompactCard
+						className="help__support-link"
+						href={ localizeUrl( 'https://wordpress.com/support/video-tutorials/' ) }
+						target="__blank"
+					>
+						<Gridicon icon="video" size={ 36 } />
+						<div className="help__support-link-section">
+							<h2 className="help__support-link-title">
+								{ this.props.translate( 'Video tutorials' ) }
+							</h2>
+							<p className="help__support-link-content">
+								{ this.props.translate(
+									'These short videos will demonstrate some of our most popular features.'
+								) }
+							</p>
+						</div>
+					</CompactCard>
+					<CompactCard
+						className="help__support-link"
+						href="https://dailypost.wordpress.com/blogging-university/"
+						target="__blank"
+					>
+						<Gridicon icon="mail" size={ 36 } />
+						<div className="help__support-link-section">
+							<h2 className="help__support-link-title">
+								{ this.props.translate( 'Email courses' ) }
+							</h2>
+							<p className="help__support-link-content">
+								{ this.props.translate(
+									'Pick from our ever-growing list of free email courses to improve your knowledge.'
+								) }
+							</p>
+						</div>
+					</CompactCard>
+					<CompactCard
+						className="help__support-link"
+						href="https://learn.wordpress.com"
+						target="__blank"
+					>
+						<Gridicon icon="list-ordered" size={ 36 } />
+						<div className="help__support-link-section">
+							<h2 className="help__support-link-title">{ this.props.translate( 'Guides' ) }</h2>
+							<p className="help__support-link-content">
+								{ this.props.translate(
+									'A step-by-step guide to getting familiar with the platform.'
+								) }
+							</p>
+						</div>
+					</CompactCard>
+				</div>
+			</>
+		);
+	};
+
+	getContactUs = () => {
+		return (
+			<>
+				<h2 className="help__section-title">{ this.props.translate( 'Contact Us' ) }</h2>
+				<CompactCard className="help__contact-us-card" href="/help/contact/">
+					<Gridicon icon="help" size={ 36 } />
+					<div className="help__contact-us-section">
+						<h3 className="help__contact-us-title">
+							{ this.props.translate( 'Contact support' ) }
+						</h3>
+						<p className="help__contact-us-content">
 							{ this.props.translate(
 								"Can't find the answer? Drop us a line and we'll lend a hand."
 							) }
 						</p>
 					</div>
-					<Button className="help__support-link-button" primary>
-						{ this.props.translate( 'Contact Us' ) }
+					<Button className="help__contact-us-button">
+						{ this.props.translate( 'Contact support' ) }
 					</Button>
 				</CompactCard>
-			</div>
+			</>
 		);
 	};
 
 	getCoursesTeaser = () => {
-		if ( ! this.props.showCoursesTeaser ) {
+		// @TODO: Find out if there are times we wouldn't want to show courses
+		/*if ( ! this.props.showCoursesTeaser ) {
```

Comment:
Same comment about removing the brace and return statement here.

Also, not sure why there's two different commenting patterns here?

Reply:
Hah, probably because I threw in that comment in a rush. 🙃